knowledge-base/pdfviewer-upload-controls-visibility.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Handle Upload Controls Visibility in TelerikPdfViewer
+description: Learn how to manage the visibility of upload controls in TelerikPdfViewer when loading a PDF file.
+type: how-to
+page_title: How to Control Upload Component Visibility in TelerikPdfViewer
```

Comment:
The word "visibility" was never used by the customer. At the same time, this word is more used in the KB than the verb simple verb "hide". May not be optimal for SEO.